student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom, :age, :name
+
+  def initialize(age, classroom = 'class one', name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky()
```

Comment:
- [ ] Kindly remove empty parenthesis after this function name as in ruby, we don't use them if there are no arguments.